### PR TITLE
[Add] Release pretrained models for QDTrack of MOT field.

### PR DESCRIPTION
**Pull request by Seerkfang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ Supported methods of multi object tracking:
 - [x] [SORT/DeepSORT](configs/mot/deepsort) (ICIP 2016/2017)
 - [x] [Tracktor](configs/mot/tracktor) (ICCV 2019)
 - [x] [ByteTrack](configs/mot/bytetrack) (arXiv 2021)
+- [x] [QDTrack](configs/mot/qdtrack) (CVPR 2021)
```
Comment (GT9505) [inline README.md:89]: move it in the front of `bytetrack`.
We need to place these methods by time.

File: README_zh-CN.md [hunk 2]
```diff
@@ -82,6 +82,7 @@ v0.11.0版本已于2022年03月04日发布，可通过查阅[更新日志](docs/
 - [x] [SORT/DeepSORT](configs/mot/deepsort) (ICIP 2016/2017)
 - [x] [Tracktor](configs/mot/tracktor) (ICCV 2019)
 - [x] [ByteTrack](configs/mot/bytetrack) (arXiv 2021)
+- [x] [QDTrack](configs/mot/qdtrack) (CVPR 2021)
```
Comment (GT9505) [inline README_zh-CN.md:85]: same here

File: configs/mot/qdtrack/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  pages={164--173},
+  year={2021}
+}
+```
+
+## Results and models on MOT17
+
+|    Method     |      Detector     |     Train Set     | Test Set | Public | Inf time (fps) | HOTA | MOTA | IDF1 | FP | FN | IDSw. | Config | Download |
+| :-------------: | :-----------------: | :------------: | :------: | :----: | :------------: | :--: | :--: | :--: |:--:|:--:| :---: | :----: | :------: |
+| QDTrack | Faster R-CNN | half-train | half-val | N     | - | 57.1 | 68.2 | 68.5 | 8373 | 42939 | 1071 | [config](qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half.py) |  [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth) &#124; [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500.log.json) |
```
Comment (GT9505) [inline configs/mot/qdtrack/README.md:34]: remeber to update the links

File: mmtrack/models/trackers/quasi_dense_embed_tracker.py [hunk 4]
```diff
@@ -200,7 +200,7 @@ def track(self, img_metas, feats, model, bboxes, labels, frame_id):
                 np.zeros((0, 6), dtype=np.float32)
                 for _ in range(model.detector.roi_head.bbox_head.num_classes)
             ]
-            return track_bboxes
+            return track_bboxes, None, None
```
Comment (GT9505) [inline mmtrack/models/trackers/quasi_dense_embed_tracker.py:203]: Follow [this](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/models/trackers/masktrack_rcnn_tracker.py#L129) to return empty track bboxes.

File: mmtrack/models/mot/qdtrack.py [hunk 5]
```diff
@@ -172,11 +172,12 @@ def simple_test(self, img, img_metas, rescale=False):
...
-            ids=track_ids,
-            num_classes=self.detector.roi_head.bbox_head.num_classes
-        )['bbox_results']
+        if track_labels is not None:
+            track_bboxes = outs2results(
+                bboxes=track_bboxes,
+                labels=track_labels,
+                ids=track_ids,
+                num_classes=self.detector.roi_head.bbox_head.num_classes
+            )['bbox_results']
```
Comment (GT9505) [inline mmtrack/models/mot/qdtrack.py:181]: Then, you can delete this `if`.

**Review by GT9505 — COMMENTED** [review]
Fogot to modify modelzoo.md in chinese and english

**Review by GT9505 — APPROVED** [review]
